CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 -Features for next release
 
+### Bug fixes
+
+- Fixed an issue where id token is not refreshed automatically (See [Issue #869](https://github.com/aws-amplify/amplify-ios/issues/869), [PR #3220](https://github.com/aws-amplify/aws-sdk-ios/pull/3220/))
```

Comment:
To provide a bit more context, would it be accurate to refine this by saying, "not refreshed automatically for customers who set short custom expiration intervals"?

Reply:
Yes, changed